services/core-web/src/styles/components/NoticeOfWork.scss

```diff
@@ -322,6 +322,11 @@
 }
 
 .contact-rows {
+  .scroll {
+    height: 1000px;
```

Comment:
We could restrict size to ~`100vh` that way it is proportional to the user's resolution?

Reply:
The height here is more about fitting an appropriate amount of data than a proportion. If it is not a concern then I can definitely change it. @mmumby

Reply:
yeah 100vh works fine, I just tested it.

Reply:
I think it should be responsive to the user's height. We may need to set this as `max-height` instead.

Reply:
what are your concerns with `height`? I've updated it to use 100vh and it's working well switching between my MacBook and monitor.

Reply:
It would be to use as much vertical real estate as possible before introducing the scrollbars. For some 4k monitors, like a 3840x2160, a cap at 2000px would be more suitable and responsive than setting it to 1000 px. Not a big deal but just trying to be more responsive based off of the user's device resolution.

Reply:
I think 75vh looked better but it's no problem.  🤓  I approved